packages/blob-reader/blob.js

```diff
@@ -16,32 +16,45 @@ async function fetchRaw({ user, repo, ref, path }) {
 }
 
 export default class Blob {
-  constructor({ el, path, lines, blobType, branch = undefined }) {
+  constructor({ el, path, lines, type, branch = undefined }) {
     this.el = el;
     this.path = path;
     this.lines = lines;
     this.branch = branch;
-    this.blobType = blobType; // one of: diffLeft, diffRight, snippet, full
+
+    if (this.lines.length > 5000) {
+      this.lines = this.lines.slice(0, 500);
```

Comment:
Is this intended to be `5000` and `500`?

Reply:
Yes it is, but I agree the numbers are confusing. The idea is to limit OctoLinker to take only the first X lines into account for files with more than X lines. I admit those are all magic numbers I just came up with. I have seen files with ~2000 lines so I thought 5000 would be a reasonable number. Since most imports are near the top I limited it further to the first 500 lines. Maybe I should update the value to 5000 as well just for the sake of clarity. Thanks for the hint!